src/components/component-rule/component-rule.module.scss

```diff
@@ -30,38 +30,38 @@
     align-items: center;
   }
 
-  &_icon {
+  &.icon {
     width: 20px;
     height: 20px;
     border-radius: var(--sb-border-radius-small);
     margin-right: var(--sb-spacing-small-medium);
     color: var(--sb-text-color-on-primary);
   }
 
-  &_description {
+  &.description {
     @include sb-content-text;
     font-weight: 400;
   }
 
-  &--recommended {
-    .vibe-sb-comps-component-rule {
-      &_icon {
+  &.recommended {
+    .componentRule {
```

Comment:
But not sure why this is needed here again... I think it can be removed

Reply:
yes